components/bl-carousel-component/component.json

```diff
@@ -13,7 +13,7 @@
3vqXC0gm6C5/uzfG5eUEZFq7Muy99iVEAalsACw3j7fDEUsAF4P2SvzV3AGM0bYQI1kwr9eYTM2ifMAY6+LlBqtH4+XAob3W3RI7UEooRVjLQ8nxMT8VjO2zwGHHjgmYw4LBaqdUKDHxUX8wIvsa1YfcIqglVi3deavMRC5jeHa/olrEsCc0Jd/TFdUEgy451t1nRLwjbMbfl2/KzzqKcssHRe/U0HEDDXLvNftBKCgClhtq3/gigLMa1YF856+5RdbmnGJj3nN96goRehoso7In2RkaOndIfvd/wBQgpdpchT8xV5WA2sFPYIBAKK8HDwcmrdSptnBQADi1HLW85azAeHRCy/QWddeIodik1Udry8uICqjSEpk14KodpV3H06hbYaFN10q2hcVBfqrrhiwGqxS1XiViom0Fpumi2CmmMVLOU2dBu2jiLorZuNY2BIiVVMrBb35WbLWN9UWUEtVY4vUEQpZYZnF5vv2gmQ3VyCjbFV4PhxmbllYqlXktdlctRsCIkaUBxeLzuzDoWRgEFI6DdvZcvIVQGWQNmC7ELLudhfVAlKNseXDKlZjC8I2HPBT/URDvCaZk8mfkjoEJqgANFOf4i4A6WgF00C6NhdVma7Diw0qFr5Ft9VY2paiXV3gavNZ/YmJNXjClK8Dng1lYXiksg3agcLQbP7oRqzYTQsFtGXDa2VL/wBnQB2GXsD6lkWCfEBeDNo3+IfixSlASjNrVmDS6hWpFUbaBc4LRfG4k4XQuS22HXHiFfaOhO2v5XwMeac2zIp+MPmAB3FrzcArGGuNbmI2SQcva9Y3K4zdHBeL+j8yx+OWAh/ml+TxLJsRNUFeeOJfcJ4N0/xj482rPaCqMCgV3r8ylqQod8Y/qUIChac436gyxYAz1dfxKlWwDV0qX8aJlzCwM85/aZgS9i4LS3qXpIqhwf4SVMtt9BrNvESA5VeQA/YkyLcJaWzZpz+0VxOltLMOn6v944uFsTXoc532ZuFnsLjw7U7RyXecUccrCFAxfhH+MEKYGCn9ifTmFzQKthUduuTzUQCMwbHdHmki4WwALYAVQGjf1iCrzL9DktXKl1l38FcZtnSvKrjNHq9w4KVLvCgW0Zv/AMg0gQy7OH1GGWBVcN5+c55KgdxaoDXF/m4dMbFvqWVSc44bso+B9RyDmDbaj+WZRw7pFlQv1SRKuCopbLz+F/MIgRR+Kcf8Q16LsaVTwKDq75lhsopa3WU/39WrVANKFTL6CEKsO91Ixe1kTDnUYrkFqM1y+oGlqoXI4J6GUM4xBoUPYetRzsVNnA0pTexfrcZSG4gCtmBZzV6SJwS45FxaQu29F9VeJtbkIcJdpdKDi95w/iggyWUy3nsJV00sHIFX3wU2dt+4TM3Q3XETXbQsMOEhXFRHbCg8AwUOqlQmtqtViWshlpq3LDmClAKBY8CCG+lVDUNIxM0G7c1Hil8wQMDs06Ko3ZwzRVI2DfS5FAIDPKhvLmoYxlMOk3BVhq0RtjIgSoFIgJaVeIAtG0bV3MbUBqHF0Sa6oRXZoAwigKC3klaJZAW7DswN9h+8ZowgVM15yArzZH6KdFoUAC8ZT6+YTQzg7XLmgwLQcnJUYVCC2nkUaMd5azUTFUSjDOR0GDIYxngiCmNBAqSqUOfG5Z14IfhL0KXa/wBoVioMXgmY1rnuFS4S5W2sb82cc9us0jQWl3ouhazXdErLTXoYAcnN5X8VVE4GWQjlgN98fxAcSYoPt9mL/iUJDKxdYonpHw6uOd1EDV1jLrP5qUgKUGw6J879yguQcKxcoYiiFdHiKWDnRTnu5uamY5TIfNF/c2sLqg26WuUoA8y0twlAVOc3p8G67cwV4BQSi35P65iN0C1H708sd4z8Fo48YbikKCPHJ9H+3MBkDAzeR55t+o41g643BBZk603+8v8Acs5zgbN0XDjzmsdWxbbAON8fUoiHSWIIbuslfEqdAYNq4yb7iIhoKKNt9/8AfiaP8a69trPX7R9d4TYcju9Rs0Mdd3f+eq92AUQFVSb/AJmEIMBdCs9Jj0zUGrS+S6tQBkAsrE6CjbaHsmkUnWCaDrY0XtfwSyWBReQTZyBWmoGtcwRFCnyncuzCQu6luYKDs1+jHp/uYdSUFPtf2gVTWH7f3BM5XloFwWfzcZiFtRQ+jw8XUS1QAo0qL/2YJYKLu7APrMKOLY0E9+Ql6D4HAAUGrdA2xLFVQ8ZV+sB8xAdQMBMVX9/HxCWv3DRK1pOCHHiGNCPZxgeca9jMhgedWhX5Rsyuwq8ocaDLmjwSmWRqlOA+mOdXKn+hFPS2I1d+41KryLEC5dNBw9eqfpAcqFBrbg115jiZWRKwNHf1jUpAIGsF+UmD9RlQLN+r+Q6lD84OSUyEAOf6yy5MaAIGGskpTNXdiFNIGs0tJxjNO1Z6ZFAOjUKEiELvKWUzB+Y00xhxV4UXFEoBzXKDmXYjLCB9r1u3TOUe8F+OCYnMOAOB8UPkx8ttZjFGfA/eMIKUuyqUF5pHoPcIaNKjAWQ40NcdQQACEXKF2Xugpu/mKc50kblDlClHL4KZQQht2hVZxTzwzCSra5nZaC6u/qo8VhsrvTp3S55ylZADcVCzKVp2El1Zg2SumxhQ2DPuoWSVDPItovGAOH3EClXVp1MbemO4VatGZoMn5Pv3G1Vk0oIAxy4HE4FJZydvk59RnaoAKCi3u7zT3DoBLJRVXVGya3YHWOBTAi1UogUtfEOoe5RXQgi0133LNO5IKoQvAkRW1UvDgnbS4cZrEzMBQkxWLe7f9UK0UkddtW8/tDHIpZ1tzRoJWjDi7cv9E9GqvzRT3cFG5Y1NrivVOoGleQq0IB7tfzFzYdZpD+cy3UNg7EtWDtxes96ur1QA8mr8017+GWY9YBzcVeMUhveTcx+nUxAtyY5gT7tBmy37HEU4zQKKXwYOf6iKF87gF2x5ftLOQK4Jy874vMdArmaBWLKunXjEtjeLlsQy+LfUoCNAHZaFv
```

Comment:
Use 3 different variation of that:
https://via.placeholder.com/400x200/000000/FFFFFF/?text=Backendless.com
Instruction here: https://placeholder.com/
Convertation service: https://www.base64-image.de/